src/NexusMods.App.UI/Pages/Sorting/LoadOrder/ILoadOrderItemModel.cs

```diff
@@ -13,4 +13,5 @@ public interface ILoadOrderItemModel : ITreeDataGridItemModel<ILoadOrderItemMode
     
     public string ModName { get; }
     public bool IsActive { get; }
+    public string SortOrdinalNumber { get; }
```

Comment:
What do you think of `DisplaySortIndex` for this?

Reply:
I was curious if there was any precedent for a string representation of an index